server/index.py

```diff
@@ -0,0 +1,22 @@
+"""Server."""
+from flask import Flask
+from mimic.text_generator import TextGenerator
+from mimic.text_generator_factory import TextGeneratorFactory
+from flask import request
+import json
+app = Flask(__name__)
+
+
+@app.route('/test-endpoint', methods=['GET'])
+def testEndpoint():
+    """Test endpoint."""
+    return "it's up!"
+
+
+@app.route('/test-model', methods=['GET'])
```

Comment:
Will this endpoint be replaced with pretrained-model-specific endpoints?

Reply:
yes, but we didn't have pretrained models when I wrote this, so I couldn't implement it.

Reply:
Cool cool cool. I was just making sure because I'm not a huge fan of the `/test-model` URL, but if it's going to be replaced that doesn't really matter.